package.json

```diff
@@ -20,7 +20,7 @@
     "@dvsa/cookie-manager": "^1.0.*",
     "chosen-npm": "^1.4.2",
     "customizr": "^1.4.0",
-    "govuk-frontend": "4.6.0",
+    "govuk-frontend": "^4.8.0",
```

Comment:
In lieu of a lockfile we should probably keep this dependency locked.

Reply:
done